packages/invoice-template/src/common/LabeledField/index.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from "react";
+import { ReactNode, FunctionComponent } from "react";
+import styled from "styled-components";
+
+type Orientation = "horizontal" | "vertical";
+
+type LabeledFieldProps = {
+    orientation: Orientation;
+    label: ReactNode;
+    value: ReactNode;
+    margin?: string;
```

Comment:
Don't like it. Gives too much freedom and whether managing margin at all is the responsibility of this component is discussable. You have two options
1. Make it a list of options e.g `"small" | "medium" | "large"`
2. Make separate component for managing margins. Something like `Spacing` in Pros.